backend/src/services/passport-strategies.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-unresolved */
```

Comment:
eslint rules should not be disabled. did you run npm run setup from the root folder ?